src/Illuminate/Container/Container.php

```diff
@@ -521,9 +521,7 @@ public function tag($abstracts, $tags)
                 $this->tags[$tag] = [];
             }
 
-            foreach ((array) $abstracts as $abstract) {
-                $this->tags[$tag][] = $abstract;
-            }
+            $this->tags[$tag] = array_merge($this->tags[$tag], (array) $abstracts);
```

Comment:
I guess it's possible to `[...$this->tags[$tag], ...(array) $abstracts]`. But I don't know what is preferred

Reply:
I agree, the spread operator is a good alternative as well.  My only preference is we swap out the `foreach`   🙏🏼